domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/pathAssertions.kt

```diff
@@ -16,3 +18,6 @@ fun <T : Path> _exists(assertionContainer: Expect<T>): Assertion =
 
 fun <T : Path> _existsNot(assertionContainer: Expect<T>): Assertion =
     ExpectImpl.builder.createDescriptive(assertionContainer, NOT_TO, RawString.create(EXIST)) { it.notExists }
+
+fun <T : Path> _parent(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, Path> =
+    ExpectImpl.feature.manualFeature(assertionContainer, DescriptionPathAssertion.PARENT) { parent }
```

Comment:
can you import `DescriptionPathAssertion.PARENT` as well please (or remove `import ch.tutteli.atrium.translations.DescriptionPathAssertion.EXIST`)

Reply:
@robstoll  Done!